app.js

```diff
@@ -280,16 +280,16 @@ app.use(function (req, res, next) {
     csrf: req.csrfToken(),
     navigation: navigation,
     gettext: req.gettext,
-    campaignHeader: app.locals.flags.campaign ? "" + (Math.floor(Math.random() * +app.locals.flags.campaign)) : 0
+    campaignHeader: app.locals.flags.campaign ? '' + (Math.floor(Math.random() * +app.locals.flags.campaign)) : 0
```

Comment:
there does not seem to be any point in even bothering with the `'' + ` part here, given that the second case is a plain number, not a string.